README.md

```diff
@@ -1 +1,136 @@
-# OOP-school-library
\ No newline at end of file
+# OOP-school-library
+
+# 📗 Table of Contents
+
+- [OOP-school-library](#OOP-school-library)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 OOP-school-library ](#-OOP-school-library-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+    - [📝 License](#-license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OOP-school-library <a name="about-project"></a>
+
+**OOP-school-library** A school library app that records what books are in the library and who borrows them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+ </details>
+ 
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://github.com/">Github</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">Gitflow</a></li>
+  </ul>
+ </details>
+ <details>
+  <summary>Frameworks</summary>
+ </details>
+<details>
+  <summary>Modules</summary>
+ </details>
+
+<details>
+  <summary>Server</summary>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
```

Comment:
- [ ] [optional] It would have been nice if you added key features of your project eg the different classes

Reply:
thank you, I'll do that